tasks/README.md

```diff
@@ -137,3 +139,26 @@ We can scale the number of testing machines in the openshift cluster with this
 command:
 
     $ oc scale rc cockpit-tasks --replicas=3
+
+## Automatic releases through a GitHub webhook
```

Comment:
I know it doesn't do that yet, but it will soon - so maybe let's already call that "GitHub webhook triggered actions"?